src/compile/data/timeunitdomain.ts

```diff
@@ -1,21 +1,21 @@
 import {Channel} from '../../channel';
 import {FieldDef} from '../../fielddef';
 import {TimeUnit} from '../../timeunit';
+import {TEMPORAL} from '../../type';
 import {extend, keys, StringSet} from '../../util';
 import {VgData} from '../../vega.schema';
 
 import {FacetModel} from './../facet';
-import {LayerModel} from './../layer';
 import {Model} from './../model';
 import {parseExpression, rawDomain} from './../time';
 
 import {DataComponent} from './data';
 
-
+// should be similar to timeUnit
```

Comment:
why? See this comment https://github.com/vega/vega-lite/pull/1304/commits/3c6a4ecfbbb6f3252972b1d7f528c3bc2924d97e#r66189916